### PR TITLE
GDScript: Add mention that `@onready @export` doesn't work as expected

### PR DESCRIPTION
**Pull request by dalexeev**
Closes https://github.com/godotengine/godot-docs/issues/7428

### CODE REVIEW

File: tutorials/scripting/gdscript/gdscript_basics.rst [hunk 1]
```diff
@@ -435,6 +438,31 @@ can replace the above code with a single line::
 
     @onready var my_label = get_node("MyLabel")
 
+.. warning::
+
+    Applying ``@onready`` and any ``@export`` annotation to the same variable
+    doesn't work as you might expect. The ``@onready`` annotation will make
```
Comment (mhilbrunner) [inline tutorials/scripting/gdscript/gdscript_basics.rst:444]:
```suggestion
    doesn't work as you might expect. The ``@onready`` annotation will cause
```

Comment (dalexeev) [reply tutorials/scripting/gdscript/gdscript_basics.rst]: I copied the text from here:

**Comment by YuriSizov** [comment]
I don't think that examples should be peppered with warning ignore annotations. It makes them messier and the need for a warning is not explained. I don't think they add to the point being demonstrated, even if they make this code runnable without errors.

**Comment by dalexeev** [comment]
> I don't think that examples should be peppered with warning ignore annotations.

In the first case, this is an example of using multiple annotations on the same variable.  Since `@onready` doesn't work with `@export`, I changed it to `@warning_ignore`. We can use abstract examples (`@annotation_a`, `@annotation_b`) instead of real examples, if you wish.

In the second case, the code demonstrates what happens when both `@onready` and `@export` are used at the same time (if the warning is disabled or ignored).

**Comment by YuriSizov** [comment]
Yes, I think the first example could be something neutral. Having a warning ignore annotation in there can be considered at face value.

For the second one having an annotation conflicts with the text where you say we don't recommend ignoring it. And people are going to come by and copy the sample as is, they always do, so giving it in this unsafe way can propagate bad coding practices. You can instead link to whatever documentation we have for suppressing warnings in the text and let the example fail if they try to use it.

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thank you! Merged.

**Pull request merged**